target/linux/mvebu/config-4.9

```diff
@@ -60,6 +60,8 @@ CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_MQ_PCI=y
 CONFIG_BOUNCE=y
+CONFIG_BTRFS_FS=y
```

Comment:
btrfs is used for the rootfs, it can't be distributed as a module. The only way around this is to use a different filesystem which will complicate the flashing process somewhat.

ext4 and F2FS are already in there just for the ClearFog, so there is some precedent. I will test the impact this has on the kernel image size.